### DISCUSSION TITLE
Quarg and Remnant

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#9249] [Q&A] [answered]
Why have there been no Quarg-Remnant interactions? The Remnant talk about most aliens (Korath, Coalition, Hai, Pug) with the exceptions being the northern aliens like the Wanderers, Bunrodea, Gegno, and the Quarg. Since the Quarg have 3 systems practically bordering the Ember Waste, it seems unlikely that they haven't had contact, so why haven't they made their presence known?

**Comment by warp-core** [comment, marked as answer]
The Quarg seem to keep a single presence in the space of each species. In their view (and the view of the Archons) the Remnant would not be a separate species, just a different faction of humanity.

Additionally, the Quarg probably aren't too interested in setting up any settlements in the Ember Waste, both because of bad memories of their own, and because the region represents the scarring left behind by Drak wars, a graveyard of sorts, which they likely avoid out of respect, unless there is some particular reason to venture there.

**Reply by SultanRTG** [reply]
Wait, what happened in the Ember Waste and Graveyard?

**Reply by warp-core** [reply]
In the Quarg's time, they encountered a species known as the Builders in what is now called the Graveyard. Following their extinction (as the result of their own creations), the Quarg and Pug had an incident which ruined the ringworld they had built in the area. You can find out more about this in game by visiting the Queri system in the Graveyard, where you will collect a mysterious item, and then asking the Quarg about it when prompted to on one of their worlds.
Much, much earlier, wars between factions of Drak resulted in severe damage to space itself, resulting in some of the weirdness of hte Ember Waste, like the unstable wormholes. The Quarg allude to this in the first contact conversation.

**Comment by Zitchas** [comment]
Warp-core covered the Quarg perspective fairly well.

From a Quarg perspective:
- Not a different race from humanity.
- General avoidance of the Ember Waste due to bad history. (Visiting the Ember Waste would probably be for the Quarg something akin to what visiting Nagasaki, Hiroshima, or Auschwitz is like for some people. A place where untold numbers of people died in horrific circumstances)
- It's unclear what, if anything, the Quarg even know about the Remnant: The Remnant are extremely secretive, make a point of hiding, their population centers are a decent ways away from Quarg systems in areas the Quarg actively avoid, and have cloaking to assist in this.
- Note: The Drak (Archons) very clearly and unambiguously do know about the Remnant. How much is unclear, but they definitely know them as a separate group from humanity who are predominantly (but not exclusively) found in the Ember Waste. The Drak don't share all they know with the Quarg, however, and there's no cause for them to bring up the Remnant in whatever general conversation they might happen to have with them. (It is unclear/unknown if there's *any* regular conversation between Quarg and Drak. The Quarg revere the Drak, and people tend to loose their reverance for things they have over much contact with, which suggests that they probably don't. No guarantees either way, though. These are two alien races who are both, for lack of a better term, NPC races. As in, the player never gets to "get inside them" and work with them in the way they do get to see inside other groups such as the Hai or Wanderers.

From a Remnant perspective:
- They are aliens with unknown goals and strategies.
- Remnant strongly value independent agency and allowing humanity to chart its own course, including technological development. So they are very suspicious of anything that looks like an alien group allowing humans to study them. What do those aliens want them to learn? Are they trying to shape humanity's development?
- These are aliens with the means, numbers, and capabilities to cause them a lot of problems. It would be trivial for the Quarg to wipe out the Remnant. As such, the Remnant need to be extremely cautious in their interactions with them, and prefer to do so via "regular human" intermediaries such as the player. See the Dawn missions as an example of these.

From an external perspective:
- The Quarg are an intrinsync part of the Coalition/Heliarch storyline. In order to avoid conflicts between the Remnant lore/story and the Coalition/Heliarch, I chose to abandon plot lines relating to the Remnant's investigation into Quarg matters or revealing anything more about what the Remnant know about them. There's enough other aliens in the galaxy, and in particular the Void Sprites, Drak, and other fauna of the Ember Waste, to provide ample investigative storylines without needing to pursue ones that resulted in conflict within the development community.

Regarding other alien races:
- The Bunrodea and Gegno are fairly new, and I haven't had the time to develop investigative strings related to them. These will probably need to wait until they have more missions and a better established character to figure out the most interesting way to write investigative missions.
- The Wanderer investigative missions are planned to be chapter 3 content, as they are harder to find and generally later in a player's game. So it's planned, and coming, and just isn't here yet.
- The Coalition/Heliarchs are probably largely going to be ignored as low priority for the Remnant storyline. Someday, perhaps, but they're not particularly relevant to the Remnant or the Ember Waste, so there won't be much, if any, content here.
- In any cases, the Remnant generally don't make themselves known to anyone. The Drak and the Korath Exiles are really the only ones who conclusively know about the Remnant, and that's not by the Remnant's choice, either.
